tokio/src/io/driver/mod.rs

```diff
@@ -274,7 +273,7 @@ cfg_not_rt! {
         /// This function panics if there is no current reactor set, or if the `rt`
         /// feature flag is not enabled.
         pub(super) fn current() -> Self {
-            panic!("there is no reactor running, must be called from the context of Tokio runtime with `rt` enabled.")
+            panic!(&format!("{} with feature `rt` enabled", crate::util::error::CONTEXT_MISSING_ERROR))
```

Comment:
Still, I mean, you can't have a runtime without enabling `rt`, but ok. This is fine with me.
```suggestion
            panic!("{} with feature `rt` enabled", crate::util::error::CONTEXT_MISSING_ERROR)
```

Reply:
@Darksonn I actually didn't think about that, switch to just `crate::util::error::CONTEXT_MISSING_ERROR` instead.